kubernetes/skyring/charts/mongodb/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
You shouldn't have to commit the `charts` directory. 
the `requirements.yml` should be all helm needs.